### DISCUSSION TITLE
Why is Haskell unchecked on the Supported Languages list?

### DISCUSSION THREAD

**Discussion by colin-heffernan** [nvim-treesitter/nvim-treesitter#3551] [Q&A] [answered]
Simply curious, as it seems to work well.

**Comment by theHamsta** [comment, marked as answer]
There is no maintainer registered for Haskell support in nvim-treesitter though there are a few contributors that helped with Haskell support.

**Reply by colin-heffernan** [reply]
Ah, makes sense. I'd step up myself, but I don't have any experience. Maybe I'll volunteer if I figure out Haskell and Treesitter enough to be comfortable.
